packages/communication-react/package.json

```diff
@@ -48,7 +48,7 @@
     "uuid": "^8.1.0"
   },
   "peerDependencies": {
-    "@azure/communication-calling": "1.13.2-beta.1 || >=1.13.1",
+    "@azure/communication-calling": "1.14.1-beta.1 || >=1.13.1",
```

Comment:
this needs to be updated in the other packlets and `common-versions.json` file as well.

Reply:
It is updated in #3182. This was missed.